ckanext/canada/helpers.py

```diff
@@ -459,3 +460,20 @@ def _add_extra_longitude_points(gjson):
     return {u'coordinates': [out], u'type': u'Polygon'}
 
 
+def recombinant_description_to_markup(text):
```

Comment:
1. it's confusing to have a helper in the canada extension with a `recombinant_...` name
2. why can't we use the markdown helper instead of this custom regexp helper?

Reply:
but I realize that I wrote this originally.. hmm.

Reply:
Hhaha yes I'm not sure who wrote this code. Maybe change it later?